contrib/fwupd.spec.in

```diff
@@ -181,6 +183,30 @@ This provides the optional package which is only required on hardware that
 can be flashed using flashrom. It is probably not required on servers.
 %endif
 
+%package qubes-dom0
+Summary: fwupd wrapper for Qubes OS - dom0 scripts
+Requires:   cabextract
+Requires:   fwupd
+Requires:   gpg
+%if 0%{?fedora} <= 25
+Requires:   python36
+%endif
```

Comment:
I don't think we care about obsolete versions

Reply:
Qubes OS R4.0 Dom0 is based on Fedora 25. If we will drop support for Qubes R4.0, Fedora 25 related lines should be deleted.

Reply:
Deleted. I've left support only for Qubes R4.1